client/src/components/RelatedItems/relatedItemsData.js

```diff
@@ -96,7 +96,14 @@ const relatedItemsdata = {
             "url": null
           }
         ]
-      }
+      },
```

Comment:
These bits of data are coming in through a separate API, correct? It might be helpful to note that down somewhere w/in your component or in your sample data (otherwise, I know I would probably forget)

Reply:
I need to change those after starting using the API. I'll just get parts of the data from Alexander and your components.